### ISSUE TITLE
add motion index readbility from ini file

### ISSUE THREAD

**Issue by redtankbai** [PredatorCZ/RevilMax#2]
motion index can be assigned from ini file that as same as you did for HavokMax, which make batch process comes possible

**Comment by PredatorCZ** [comment]
Isn't this a duplicate of #1?

**Comment by redtankbai** [comment]
Yes, but #1 been closed and you still request issue task when i mention this in HavokMax, so i have to make a other one

**Comment by PredatorCZ** [comment]
You know, you can reopen issues afaik.

**Comment by redtankbai** [comment]
it seems i can only reopen the issue which closed by myself, #1 doesn't fit the condition
https://stackoverflow.com/questions/21333654/how-to-re-open-an-issue-in-github

**Comment by PredatorCZ** [comment]
k, I will delet this, and reopen #1.
